frontend/src/app/transactions/transactions.component.html

```diff
@@ -0,0 +1,10 @@
+<table *ngIf="transactions != undefined">
+  <tr *ngFor="let transaction of transactions">
+    <td>{{transaction.id}}</td>
+    <td>{{transaction.accountNumber}}</td>
+    <td>{{transaction.direction}}</td>
+    <td>{{transaction.amount}}</td>
+  </tr>
+</table>
+
+<p *ngIf="error">Some error while fetching transactions</p>
```

Comment:
Use else. After another successful attempt this error statement would still be visible.